File: pset6/mario/less/mario.py
from cs50 import get_int
import sys
while True:
    height = get_int("Height:")
    if height < 9 and height > 0:
        break
for i in range(height):
    num_hash = i + 1
    num_spaces = height - num_hash
    print(" " * num_spaces, end="")
    print("#" * num_hash)

File: pset1/mario/mario.c
#include<stdio.h>
#include<cs50.h>

int main(void)
{
    int n, i, j;
    do
    {
        n = get_int("Enter height:");
    }
    while (n < 1 || n > 8);
    for (i = 1; i <= n; i++)
    {   for (j = n; j > i; j--)
        printf(" ");
        j=j-1;
        for (j = 1; j <= i; j++)
        printf("#");
        printf("\n");
    }

}